### ISSUE TITLE
seems not suport set format to function

### ISSUE THREAD

**Issue by zhuzhuppx** [charliekassel/vuejs-datepicker#470]
vue.esm.js?efeb:578 [Vue warn]: Invalid prop: type check failed for prop "format". Expected String, got Function.

**Comment by zhuzhuppx** [comment]
when i write like this,it works……，but your api is wirte in methods!
`	computed: {
			customFormatter(date) {
				return moment(date).format(this.format || 'YYYY-MM-DD');
			},
		},`

**Comment by zyfcjtc** [comment]
@zhuzhuppx 
```
<datepicker :format="customFormatter(date)" v-model="date"></datepicker>
```
I had the same issue before, and this would fix the problem.

the one you provide only works if there is only one date picker

**Comment by zhuzhuppx** [comment]
ok ,i see,thank you

**Comment by charliekassel** [comment]
Thank you for letting me know.

Fixed in 19f16fc and released in v1.0.4
